src/AzureAuth.Test/CommandMainTest.cs

```diff
@@ -342,12 +342,60 @@ public void TestEvaluateOptionsWithoutAliasMissingRequiredOptions()
             subject.EvaluateOptions().Should().BeFalse();
             this.logTarget.Logs.Should().Contain(new[]
             {
-                "The --resource field is required.",
+                "The --resource field or the --scope field is required.",
                 "The --client field is required.",
                 "The --tenant field is required.",
             });
         }
 
+        /// <summary>
+        /// The test to evaluate options without resource but with scopes.
+        /// </summary>
+        [Test]
+        public void TestEvaluateOptionsWithOverridedScopes()
+        {
+            CommandMain subject = this.serviceProvider.GetService<CommandMain>();
+            subject.Resource = null;
+            subject.Client = "e19f71ed-3b14-448d-9346-9eff9753646b";
+            subject.Tenant = "9f6227ee-3d14-473e-8bed-1281171ef8c9";
+            subject.Scopes = new string[] { ".default" };

```

Comment:
From a raw data perspective this works, however - this is would not work for real.

Here you would need to pass `<resource ID/URL>/.default`. We add the `/.default` by default if only passing the resource. But if passing the scope, you have to pass the full scope. 

While this doesn't impact this test functionally, it does impact it from a documentation perspective.